app/views/events/_apply_button.erb

```diff
@@ -0,0 +1,7 @@
+<% if current_user.present? && current_user.application_letter_for_event?(event) %>
+    <%= link_to t("helpers.links.show_application"), check_application_letter_path(current_user.application_letter_for_event(event)),
+                :class => 'btn btn-default btn-sm' %>
+<% elsif not event.after_deadline? %>
```

Comment:
Thanks for the hint. But is it really necessary? I think if a person can not apply there should not even be an option to create an application letter for them?